newsfragments/45961.significant.rst

```diff
@@ -0,0 +1,12 @@
+Replace ``external_trigger`` check with ``DagRunType``
```

Comment:
We should mention if the user currently relies on `external_trigger`, they can replace it by checking `run_type`.